.github/workflows/docker-build-test.yaml

```diff
@@ -187,7 +187,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: testnet
+      IMAGE_TAG: f659013179a5bb8e6cfa629426aa78f8f43a78b6 # pin the base image tag to this commit on testnet
```

Comment:
This is nice and all but this also means we have to remember to update this every time we do a LLT release @sherry-x

Reply:
> This is nice and all but this also means we have to remember to update this every time we do a LLT release @sherry-x

Yes, but this update is a necessary test step, because it will require to run the compat test with modified testnet against same other code.

Reply:
Stopping compat test for now until we identify the root cause: https://github.com/aptos-labs/aptos-core/pull/4289

We should definitely pin the git rev for the branch cut, but will not do so until we figure out what broke compat

Reply:
I think we need to have a story around ultimately updating this that is not manual because we're going to forget to update this and then we're going to realize we're broken at which point a lot of the value of this test is lost